package.json

```diff
@@ -5,9 +5,10 @@
     "scripts": {
         "dev": "next dev",
         "build": "next build",
-        "start": "next start",
+        "start": "serve out",
```

Comment:
`next start` is for dynamic sites
As we use static sites, we can just `serve` the build output